src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxNode.vb

```diff
@@ -83,20 +83,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             End Get
         End Property
 
-        Protected Overrides Function GetSlotCount() As Integer
-            Throw ExceptionUtilities.Unreachable
-        End Function
-
-        Protected Property _slotCount As Integer
-            Get
-                Return Me.SlotCount
-            End Get
-
-            Set(value As Integer)
-                Me.SlotCount = value
-            End Set
-        End Property
```

Comment:
such a bizarre property.  it existed just to forward to SlotCount... so i've removed it entirely.